examples/simple/simple.mk

```
@@ -0,0 +1,91 @@
+QEMU := qemu-system-aarch64
+
+MICROKIT_TOOL ?= $(MICROKIT_SDK)/bin/microkit
+
+BOARD_DIR := $(MICROKIT_SDK)/board/$(MICROKIT_BOARD)/$(MICROKIT_CONFIG)
+SYSTEM_DIR := ${EXAMPLE_DIR}/board/$(MICROKIT_BOARD)
+SYSTEM_FILE := ${SYSTEM_DIR}/simple.system
+IMAGE_FILE := loader.img
+REPORT_FILE := report.txt
+
+vpath %.c ${VMM} ${EXAMPLE_DIR}
+
+IMAGES := vmm.elf
+
+CFLAGS := \
+	  -mstrict-align \
+	  -ffreestanding \
+	  -g3 -O3 -Wall \
+	  -Wno-unused-function \
+	  -DMICROKIT_CONFIG_$(MICROKIT_CONFIG) \
+	  -DBOARD_$(MICROKIT_BOARD) \
+	  -DCONFIG_$(MICROKIT_CONFIG) \
+	  -I$(BOARD_DIR)/include \
+	  -I${VMM}/src \
+	  -I${VMM}/src/arch/aarch64 \
+	  -I${VMM}/src/util \
```

```suggestion
	  -I${LIBVMM}/include \
```

For once you rebase.

---

Done